tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe("django-ory-cloud", () => {
+  for (const app of [
+    { url: "http://localhost:4000/", name: "django" },
+  ]) {
+    test.describe(app.name, async () => {
+      test("able to use login and sign up", async ({ page }) => {
+        await page.goto(app.url, { waitUntil: "networkidle" })
```

Comment:
I suggest that you configure the host in `playwright.config.ts` and then indicate here that you go to the _homepage_ of your app.

```suggestion
        await page.goto("/", { waitUntil: "networkidle" })
```